hanadb_exporter/db_manager.py

```diff
@@ -125,6 +138,12 @@ def start(self, host, port, **kwargs):
                 if 'Invalid value for KEY' in str(err):
                     raise hdb_connector.connectors.base_connector.ConnectionError(
                         'provided userkey is not valid. Check if dbapi is installed correctly')
+
+                if is_scale_out_mode:
+                    self._logger.info('System database is not accessible on this host, scale-out is enabled, checking other hosts')
+                    self._verify_sys_db_on_other_hosts(port,  kwargs.get('instance_number', '00'), connection_data)
+                    time.sleep(SCALE_OUT_INTERVAL)
```

Comment:
We cannot have this `sleep` here, it would stall all the execution. Keep in mind that this is executed in a http call that we cannot keep alive that long

Reply:
Thanks, I am changing that.